docs/earthfile/builtin-args.md

```diff
@@ -36,6 +36,8 @@ The following builtin args are available
 | `EARTHLY_GIT_ORIGIN_URL` | The git URL detected within the build context directory. If no git directory is detected, then the value is an empty string. Please note that this may be inconsistent, depending on whether an HTTPS or SSH URL was used. | `git@github.com:bar/buz.git` or `https://github.com/bar/buz.git` |
 | `EARTHLY_GIT_PROJECT_NAME` | The git project name from within the git URL detected within the build context directory. If no git directory is detected, then the value is an empty string. | `bar/buz` |
 | `EARTHLY_GIT_COMMIT_TIMESTAMP` | The timestamp, as unix seconds, of the git commit detected within the build context directory. If no git directory is detected, then the value is an empty string. | `1626881847` |
+| `EARTHLY_VERSION` | The version of Earthly currently running. | `v0.6.2`, `dev-main` |
+| `EARTHLY_BUILD_SHA` | The git hash of the commit which built the currently running version of Earthly. | `1a9eda7a83af0e2ec122720e93ff6dbe9231fc0c` |
```

Comment:
Mind splitting the docs changes into another PR that branches off of `next`?

Reply:
Done! https://github.com/earthly/earthly/tree/brandon/docs-builtin-args